concourse/pipeline.yml

```diff
@@ -131,6 +150,11 @@ jobs:
   - name: deploy-app-staging
     serial: true
     plan:
+      - try:
```

Comment:
[Docs recommend ](https://github.com/alphagov/paas-grafana-annotation-resource#notes)we use try to prevent this blocking deploys, seemed sensible!